packages/@vuepress/theme-default/components/Page.vue

```diff
@@ -146,8 +140,8 @@ export default {
         return (
           base.replace(endingSlashRE, '')
            + `/src`
-           + `/${docsBranch}`
-           + (docsDir ? '/' + docsDir.replace(endingSlashRE, '') : '')
+           + `/${docsBranch}/`
+           + (docsDir ? docsDir.replace(endingSlashRE, '') + '/' : '')
```

Comment:
It seems that changes here is unnecessary?

Reply:
In past implementations, `path` is prefixed with a `/`, however in the pr, `path` is not prefixed with `/`. So adding a ending slash is a must.